backend/capellacollab/projects/users/routes.py

```diff
@@ -61,6 +62,22 @@ def check_user_not_in_project(project: DatabaseProject, user: DatabaseUser):
         )
 
 
+@router.get("/current", response_model=ProjectUser)
+def get_current_user(
+    user: DatabaseUser = Depends(get_own_user),
+    project: DatabaseProject = Depends(get_existing_project),
+    db: Session = Depends(get_db),
+    token: JWTBearer = Depends(JWTBearer()),
+) -> ProjectUserAssociation:
+    if RoleVerification(required_role=Role.ADMIN, verify=False)(token, db):
+        return ProjectUser(
```

Comment:
The type signature doesn’t match all the returned types.

Reply:
Fixed.